### DISCUSSION TITLE
Is `render` function support Vercel Edge Runtime?

### DISCUSSION THREAD

**Discussion by ysm-dev** [resend/react-email#579] [Q&A]
Is `render` function support Vercel Edge Runtime?

**Comment by zenorocha** [comment]
What’s the use case for running render on Vercel’s Edge Runtime?

**Comment by ysm-dev** [comment]
Generate html string from React component from edge runtime. (or cloudflare workers)

**Comment by bukinoshita** [comment]
@ysm-dev Can you try testing? We didn't use with Edge yet, would love to see the results.

**Comment by ysm-dev** [comment]
@bukinoshita 
`render` function doesn't work on Edge runtime, so I just used serverless functions.

Maybe I can just renderToString in browser(client-side), and send it to edge function.
(If there is no security problem)

**Comment by nickbytes** [comment]
Also curious about this. Looking into running on a scheduled Cloudflare Worker, will report back if I can make this work.

**Comment by KyGuy2002** [comment]
I am trying to use this with cloudflare pages functions and my worker fails to deploy with `Error: Script startup exceeded CPU time limit.` when I have any react-email components within my render method.
